### PR TITLE
Hot fixes and cypress test changes

### PR DESCRIPTION
**Pull request by mengweieric**
### Description
This branch was used in a demo environment for reinvent, therefore it contains couple of different hot fixes. It also includes changes:

- Skipped some cypress tests that are testing against unreleased visualizations, and those that needs to be rewritten and improved
- Resolved rest of failing cypress tests of event analytics 

This PR is for updating 2.x with latest fixes, event analytics refactoring, and updating PR #1304 to be backported to main.

### Issues Resolved
#1272 
#1274 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ps48 — DISMISSED** [review]

File: dashboards-observability/public/components/event_analytics/explorer/explorer.tsx [hunk 1]
```diff
@@ -207,11 +207,11 @@ export const Explorer = ({
   liveTailTabIdRef.current = liveTailTabId;
   liveTailNameRef.current = liveTailName;
 
+  let minInterval = 'y';
```
Comment (derek-ho) [inline dashboards-observability/public/components/event_analytics/explorer/explorer.tsx:210]: I had already done the minInterval fix in a separate PR: 
https://github.com/opensearch-project/observability/pull/1298
so I don't think this is necessary anymore?

Comment (mengweieric) [reply dashboards-observability/public/components/event_analytics/explorer/explorer.tsx]: that's probably changed from resolving conflicts, thanks for catching this.

**Review by derek-ho — APPROVED** [review]
LGTM - not too familiar with cypress or viz code yet

**Review by ps48 — APPROVED** [review]
